mmv1/third_party/terraform/.teamcity/components/get_services.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import generated.ServicesListGa
+import generated.ServicesListBeta
+
+// This file is maintained in the GoogleCloudPlatform/magic-modules repository and copied into the downstream provider repositories. Any changes to this file in the downstream will be overwritten.
+
+fun getServicesList(Services: Array<String>, version: String): Map<String,Map<String,String>> {
+    if (Services.isEmpty()) {
+        throw Exception("No services found for version $version")
+    }
+
+    var servicesList = mutableMapOf<String,Map<String,String>>()
+    for (service in Services) {
+        if (version == "GA" || version == "GA-MM") {
+            servicesList[service] = ServicesListGa.getOrElse(service) { throw Exception("Service $service not found") }
+        } else if (version == "Beta" || version == "Beta-MM") {
+            servicesList[service] = ServicesListBeta.getOrElse(service) { throw Exception("Service $service not found") }
+        } else {
+            throw Exception("Invalid version $version")
+        }
+    }
+
+    when (version) {
+        "GA" -> return servicesList
+        "Beta" -> {
+            servicesList = servicesList.mapValues { (_, value) ->
+                value + mapOf(
+                    "displayName" to "${value["displayName"]} - Beta",
+                    "path" to (value["path"]?.replace("./google/", "./google-beta/") ?: "")
```

Comment:
Is this transform necessary? I thought that when the version is "Beta" then information in [inputs/services_beta.kt](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt) would be used, which should be maintained to have beta-specific paths

Reply:
the `transformation of path` is unnecessary and was removed in the latest commit. 

The `displayName` is still needed since without explcitly setting the displayName to include beta/MM we get an error due to duplicate name usage.

Reply:
That's fine, it was only the transform of `path` that I was interested in. I get the need to alter the names- I've suffered the pain of TeamCity's constraints around name uniqueness despite being at _different levels of nesting_ 😭